frontend21/scss/components/extra/about/components-showcase.scss

```diff
@@ -1,23 +1,139 @@
-.#{project('components-showcase')} {
-  background: red;
+.#{project("components-showcase")} {
+	position: relative;
+	height: 100%;
 
-  &-buttons {
-    background: lime;
-  }
+	&-buttons {
+		display: grid;
+		grid-template-columns: repeat(2, 1fr);
+		grid-gap: 0.625rem;
+		margin-bottom: 1.875rem;
+		button {
+			margin-bottom: 0;
+		}
+	}
 
-  &-demos {
-    background: cyan;
-  }
+	&-demos {
+		.show {
+			position: relative;
+			display: block;
+			height: 455px;
+		}
 
-  &-demo {
-    background: yellow;
+		.hide {
+			display: none;
+		}
```

Comment:
Style nit: Move those to the `&-demo` block and use `&.--show` as selectors instead.